### PR TITLE
chore: update readme

### LINKED ISSUE #47: Update README with latest information.

### PR DESCRIPTION
**Pull request by Akarshit**
Closes: #47 
This PR updates the README adding explanation about the libraries used and their workings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,24 +6,31 @@
...
-### How it works
+## How it works
 
 This plugin registers the schema and resolvers that are required by account-js.
 
-Whenever a header with name `Authorization` is present in the request, this plugin uses account-js to get the user for that token. Then the user is attached to the `req` object.
+A express middleware is added whose job is to extract the `Authorization` header from the request and uses account-js to get the user for that token. The format should be `Authorization: Bearer <YOUR_ACCESS_TOKEN>`. Then the user is attached to the `req` object.
 
-Also on each request, `accountsGraphQL.context` is called so that it can pass the `user` to the "internal context" of account-js.
+A function is added to `functionByType.graphQLContext` which links the graphQL context used in the api-core with the "internal context" used by account-js. This is needed because account-js also keeps tracks of the tokens and uses a separate GraphlQL server.
```
Comment (delagroove) [inline README.md]: extra l

**Review by delagroove — COMMENTED** [review]
just 1 typo

File: README.md (hunk 1 above)
Comment (delagroove) [reply README.md]: in graphlQL

**Comment by Akarshit** [comment]
Updated.

**Review by delagroove — APPROVED** [review]

**Pull request merged**